### PR TITLE
Fix Grammar - Closes #422 

### LINKED ISSUE #422: Grammatical errors inside Core Docs
**Describe the bug**

To ensure all English text, punctuation, spelling and grammar are correct.

### PR DESCRIPTION
**Pull request by chris529**
Closes #422

### CODE REVIEW

File: modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ There are three available distributions of Lisk Core which can be seen below:
 === Binary
 
 The *default* way to setup Lisk Core.
-The xref:{url_binary_installation}[Binary installation] is an easy and automated way to set up Lisk Core, which includes almost completed automated update scripts; together with a selection of tools to assist with seamlessly maintaining a Lisk Node.
+The xref:{url_binary_installation}[Binary installation] is an easy and automated way to set up Lisk Core, which includes automated update scripts that are almost complete, together with a selection of tools to assist with seamlessly maintaining a Lisk Node.
```
Comment (Tschakki) [inline modules/ROOT/pages/getting-started.adoc]:
```suggestion
The xref:{url_binary_installation}[Binary installation] is an easy and automated way to set up Lisk Core, which includes almost completely automated update scripts, together with a selection of tools to assist with seamlessly maintaining a Lisk Node.
```

File: modules/ROOT/pages/reference/config.adoc [hunk 2]
```diff
@@ -338,8 +338,7 @@ A breakdown of the code structure can be seen below:
 <10> Milestone 1.
 <11> Milestone 2.
 <12> Milestone 3.
-<13> Milestone 4.
-<14> Start rewards at block (n).
+<13> Milestone 4. <14> Start rewards at block (n).
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/config.adoc]: Format issues again. Also for the lines below.
```suggestion
<13> Milestone 4. 
<14> Start rewards at block (n).
```

File: modules/ROOT/pages/setup/binary.adoc [hunk 3]
```diff
@@ -98,33 +98,35 @@ bash installLisk.sh install -r test
...
-This will be much faster than synching from the genesis block.
+This will be much faster than synchronizing from the genesis block.
 
 The installation may take a few minutes.
-Check the output of the script to verify that the installation was successful.
+Please check the output of the script to verify that the installation was successful.
 
-If you recognize an error, try to resolve it by analyzing the error output, otherwise, you can have a look at our xref:{url_troubleshooting}[Troubleshooting Section].
+If an error appears, try to resolve it by analyzing the error output.
+If this is not possible, please refer to the xref:{url_troubleshooting}[Troubleshooting Section].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/binary.adoc]: The Troubleshooting page got removed, we should refer to the new dev forum here
```suggestion
If this is not possible, please refer to the xref:{url_developers_forum}[Developer's Forum].
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/update/docker.adoc [hunk 4]
```diff
@@ -10,26 +10,26 @@ Mona Bärenfänger <mona@lightcurve.io>
 [[minor-version]]
 == Upgrade minor versions
 
-To upgrade a Lisk installation using Docker Compose you simply need to export the new version number in `ENV_LISK_VERSION`.
+To updatee a Lisk installation using Docker Compose it is only required to export the new version number in `ENV_LISK_VERSION`.
```
Comment (Tschakki) [inline modules/ROOT/pages/update/docker.adoc]:
```suggestion
To update a Lisk installation using Docker Compose it is only required to export the new version number in `ENV_LISK_VERSION`.
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 5]
```diff
@@ -94,7 +94,7 @@ lisk core:status lisk-testnet
 --
 ====
 
-TIP: See for all available options the Lisk Commander xref:{url_sdk_commander_liskcore}[Command reference for Lisk Core] as well as the xref:{url_sdk_commander_commands}[general Command reference].
+TIP: All available options for the Lisk Commander can be seen in the xref:{url_sdk_commander_liskcore}[Command reference for Lisk Core] as well as the xref:{url_sdk_commander_commands}[general Command reference].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]:
```suggestion
TIP: All available options can be seen in the xref:{url_sdk_commander_liskcore}[Lisk Commander Command reference for Lisk Core] as well as the xref:{url_sdk_commander_commands}[general Command reference].
```

File: modules/ROOT/pages/reference/config.adoc [hunk 6]
```diff
@@ -408,8 +408,7 @@ Defaults to all domains.
 <70> How many transactions can be included in a single bundle.
 <71> Specifies how many times a transaction broadcast from the node will be relayed.
 <72> Sets the maximum size of each transaction queue.
-Default: 1000
-<73> Contains forging options for delegates.
+Default: 1000 <73> Contains forging options for delegates.
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/config.adoc]: Format broken

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/reference/config.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/reference/config.adoc]: Think it should be fixed now..

**Review by Tschakki — APPROVED** [review]

**Pull request merged**